src/controllers/courses-controller.js

```diff
@@ -0,0 +1,58 @@
+import courses from '../models/courses.json' assert { type: "json" };
```

Comment:
Usually, using experimental features is not a good idea for a project. Ok for testing, but if possible it's best to avoid such things until they are became stable. As for me, better to remake this to simple js file.

`import courses from "../models/courses.js";`